<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBIFWidget</class>
 <widget class="QFrame" name="CBIFWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIF Widget</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="toggleButtons">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="skipBackwardToggleBtn">
         <property name="toolTip">
          <string>Skip Backward</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/skipbackward.png</normaloff>:/BIFPlayerResources/skipbackward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prevToggleBtn">
         <property name="toolTip">
          <string>Previous Frame</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/prev.png</normaloff>:/BIFPlayerResources/prev.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="playPauseBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/play.png</normaloff>:/BIFPlayerResources/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="nextToggleBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/next.png</normaloff>:/BIFPlayerResources/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skipForwardToggleBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/skipforward.png</normaloff>:/BIFPlayerResources/skipforward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="discreteButtons">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>224</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="skipBackwardDiscreteBtn">
         <property name="toolTip">
          <string>Skip Backward</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/skipbackward.png</normaloff>:/BIFPlayerResources/skipbackward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prevDiscreteBtn">
         <property name="toolTip">
          <string>Previous Frame</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/prev.png</normaloff>:/BIFPlayerResources/prev.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="playBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/play.png</normaloff>:/BIFPlayerResources/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pauseBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/pause.png</normaloff>:/BIFPlayerResources/pause.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="nextDiscreteBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/next.png</normaloff>:/BIFPlayerResources/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skipForwardDiscreteBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="BIFPlayerResources/BIFPlayerResources.qrc">
           <normaloff>:/BIFPlayerResources/skipforward.png</normaloff>:/BIFPlayerResources/skipforward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>224</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="BIFPlayerResources/BIFPlayerResources.qrc"/>
 </resources>
 <connections/>
</ui>
